### ISSUE TITLE
LanguageServer should not publish ignored issues

### ISSUE THREAD

**Issue by akosyakov** [eclipse/xtext-core#453]
Right now ignores issues are published as hints
https://github.com/eclipse/xtext-core/blob/ffca82299acca8543fecbc0a41126ea1b986c506/org.eclipse.xtext.ide/src/org/eclipse/xtext/ide/server/LanguageServerImpl.xtend#L286-L291

**Comment by cdietrich** [comment]
ignored issues should never be created

```
org.eclipse.xtext.validation.AbstractDeclarativeValidator.addIssue(String, EObject, EStructuralFeature, int, String, String...)
``` 

can you give a hint how you managed to get one @akosyakov ?

**Comment by cdietrich** [comment]
fixed in 2.13. would still want to provide a test therefore hints on reproduction welcome
